### ISSUE TITLE
Binary exporter

### ISSUE THREAD

**Issue by inaimathi** [inaimathi/cl-notebook#30]
An exporter that generates binary of the given notebook. Options include `buildapp` and [electron](https://electronjs.org/)

**Comment by vindarel** [comment]
Hello, maybe a link to [Ceramic](https://github.com/ceramic/ceramic) ? Also isn't [`asdf:make`](https://lispcookbook.github.io/cl-cookbook/scripting.html#with-asdf) as capable as buildapp and simpler now ?

**Comment by inaimathi** [comment]
Yup. These approaches would also make self for publishing binaries of `cl-notebook` itself.
